src/css/horizontal-image.scss

```
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.bb-horizontal-image {
+    text-align: center;
+    padding: $layout-xs $layout-md 0 $layout-md;
```

It doesn't entirely matter, but since left and right padding is the same these could all use the three-value shorthand format `top left/right bottom`:

```suggestion
    padding: $layout-xs $layout-md 0;
```